modules/volume/pulseaudio/pulse.go

```diff
@@ -25,13 +25,25 @@ import (
 	"github.com/godbus/dbus/v5"
 )
 
+type deviceType int
+
+const (
+	SinkDevice deviceType = iota // Sinks (audio outputs, e.g. headphones)
```

Comment:
Can you move the comments to be godoc style? (Something like `// SinkDevice represents audio outputs, e.g. headphones` on the line above instead of after).